### PR TITLE
Us 8 add interviewers

### PR DESCRIPTION
**Pull request by mateuszs-kainos**

### CODE REVIEW

File: src/main/java/resources/PersonResources.java [hunk 1]
```diff
@@ -27,15 +24,32 @@
...
+                                 @FormParam("firstname") @NotEmpty String firstName,
+                                 @FormParam("lastname") @NotEmpty String lastName,
+                                 @FormParam("isdev") Boolean isDev,
+                                 @FormParam("istest") Boolean isTest,
+                                 @FormParam("isweb") Boolean isWeb,
+                                 @FormParam("bandlevel") Integer bandLevel) {
+        Persons person = new Persons(email, firstName, lastName, isDev, isTest, isWeb, bandLevel);
+        personsDao.create(person);
+
+        return "Interviewer created";
```
Comment (wojciechh-kainos) [inline src/main/java/resources/PersonResources.java]: We should return the created person here

**Comment by RogowskaMalgorzata** [comment]
All good.

File: src/main/java/resources/SlotsResource.java [hunk 2]
```diff
@@ -1,47 +1,66 @@
...
+                           @PathParam("person_id") long person_id,
+                           @PathParam("date_from") String date_from,
+                           @PathParam("date_to") String date_to) throws ParseException {
+
+        SimpleDateFormat formatter = new SimpleDateFormat("dd-MM-yyyy");
+
+        Date start = formatter.parse(date_from);
+        Date end = formatter.parse(date_to);
+
+        Persons person = personsDao.getById(person_id);
```
Comment (wojciechh-kainos) [inline src/main/java/resources/SlotsResource.java]: seems strange that we need to get person out of the db just to set it here

File: src/main/resources/assets/js/application/recruiter/tdprRecruiterModuleConfig.js [hunk 3]
```diff
@@ -21,6 +21,13 @@ define(['angular'
                     controller: "tdprWeekTableController"
                 }
             }
+        }).state("tdpr.recruiter.add", {
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/tdprRecruiterModuleConfig.js]: this state name suggests its purpose is to add recruiters. Maybe change that to addInterviewer?

File: src/main/resources/assets/js/application/recruiter/tdprRecruiterModuleConfig.js [hunk 4]
```diff
@@ -21,6 +21,13 @@ define(['angular'
...
+            url: "/add",
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/tdprRecruiterModuleConfig.js]: same here - "add-interviewer"

File: src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    <br>
+
+    Is tester?
+    <select name="istest">
+        <option value="true">true</option>
+        <option value="false">false</option>
+    </select>
+    <br>
+
+    Is web ops?
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html]: Maybe just make it a checkbox?

File: src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+    Is developer?
+    <select name="isdev">
+        <option value="true">true</option>
+        <option value="false">false</option>
+    </select>
+    <br>
+
+    Is tester?
+    <select name="istest">
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html]: Checkbox maybe?

File: src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+<form action="api/person/create" method="post">
+    Email:<br>
+    <input type="text" name="email"><br>
+    First name:<br>
+    <input type="text" name="firstname"><br>
+    Last name:<br>
+    <input type="text" name="lastname"><br><br>
+
+    Is developer?
+    <select name="isdev">
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html]: Why use select for a simple checkbox functionality?

File: src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html [hunk 8]
```diff
@@ -0,0 +1,24 @@
+<h1>Add new interviewer</h1>
+
+<div id="errorAlert" ng-show="error">{{error}}</div>
+<form class="simple-form">
+E-mail: <input type="email" ng-model="person.email" required /><br />
+First name: <input type="text" ng-model="person.firstName" required /><br />
+Last name: <input type="text" ng-model="person.lastName" required /><br />
+
+Deweloper?: <input type="radio" ng-model="person.isDev" value="true" ng-required="!person.isDev" /> tak
```
Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html]: Developer*

Comment (wojciechh-kainos) [inline src/main/resources/assets/js/application/recruiter/views/tdpr-recruiter-add-interviewer.html]: Remove ? signs maybe?

**Pull request merged**